### DISCUSSION TITLE
Code block content coming from another file

### DISCUSSION THREAD

**Discussion by dodalovicgran** [facebook/docusaurus#8750] [Q&A] [answered]
Hi, I'm having the following structure:

![image](https://user-images.githubusercontent.com/116893230/223944623-352a0b15-e143-4eec-a485-ae04084e00b4.png)

What I would like to achieve is that in the `document-management.md` file I have something like:

![image](https://user-images.githubusercontent.com/116893230/223945313-fea7eae9-afa1-4950-bd55-e5d80b16ccff.png)

Markdown should then render the value coming from the `yaml` file.

Is there such a possibility?

Thank You for the wonderful tool!

**Comment by slorber** [comment, marked as answer]
This should work:
https://docusaurus.io/docs/markdown-features/react#importing-code-snippets

proprietary webpack feature so use it wisely

**Reply by dodalovicgran** [reply]
Wonderful, thank You!!!!!
